### PR TITLE
feat: add readme

### PR DESCRIPTION
**Pull request by Zzhiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,194 @@
...
+
+English | [中文](README_cn.md)
+
+Kitexcall is a command-line tool for sending JSON general requests using kitex, similar to how curl is used for HTTP.
+
+## Features
+
+- **Supports Thrift/Protobuf:** It supports IDL in Thrift/Protobuf formats.
+- **Supports Multiple Transport Protocols:** It supports transport protocols like Buffered, TTHeader, Framed, and TTHeaderFramed, with plans to support GRPC (Protobuf and Thrift Streaming) in the future.
+- **Supports Common Client Options:** It allows specify common client options, such as client.WithHostPorts, etc.
```
Comment (felix021) [inline README.md:11]: Format no working. Please review first in your own repo: https://github.com/Zzhiter/kitexcall/tree/add_readme

File: README.md [hunk 2]
```diff
@@ -1 +1,194 @@
...
+## Usage
+
+### Basic Usage
+
+When using the kitexcall tool, you need to specify several required arguments, including the path to the IDL file, the method name, and the data to be sent. Example:
+
+- Directly specifying request data:
+
+```bash
+kitexcall -idl-path echo.thrift -m echo -d '{"message": "hello"}' -e 127.0.0.1:9999
```
Comment (felix021) [inline README.md:32]: pls separate the command and its output to make it clearer

```bash
kitexcall -idl-path echo.thrift -m echo -d '{"message": "hello"}' -e 127.0.0.1:9999
```
Output:
> [Status]: Success
> {
>     "message": "hello"
> }

File: README.md [hunk 3]
```diff
@@ -1 +1,194 @@
...
+[Status]: Success
+{
+    "message": "hello"
+}
+```
+
+- Or reading request data from a file:
+
+```bash
+kitexcall -idl-path echo.thrift -m echo -d '{"message": "hello"}' -e 127.0.0.1:9999 -f input.json
```
Comment (felix021) [inline README.md:42]: `-d` not removed?

Comment (felix021) [reply README.md:42]: And pls rearrange the example:

1. Place the IDL at the beginning of this Example
2. Instructions on creating a file `input.json` with the contents before showing them the command

Comment (Zzhiter) [reply README.md]: All fixed

**Review by felix021 — APPROVED** [review]

**Pull request merged**